pkg/aws/services/acm/acm.go

```diff
@@ -0,0 +1,179 @@
+package acm
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/acm"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/irsa-operator/pkg/aws/services/route53"
+	"github.com/giantswarm/irsa-operator/pkg/key"
+	"github.com/giantswarm/irsa-operator/pkg/util"
+)
+
+func (s *Service) EnsureCertificate(domain string, customerTags map[string]string) (*string, error) {
+	s.scope.Info(fmt.Sprintf("Ensuring ACM certificate for domain %q", domain))
+
+	// Check if certificate exists
+	existing, err := s.findCertificateForDomain(domain)
+	if err != nil {
+		if err != nil {
```

Comment:
Should only be one if != nil 
```suggestion
	if err != nil {
		if err != nil {
```

Reply:
oh rly? :smile:

Reply:
fixed thx